.github/dependabot.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  reviewers:
+  - jglick
+  - oleg-nenashev
+  - batmat
```

Comment:
```suggestion
```

Should we remove that in favor of CODEOWNERS @jglick @batmat ? I'd guess so

Reply:
If that has a similar effect, then sure!